hrls will not start after upgrade to version 8 of mysql-connector-java

In #53 the mysql-connector-java version is updated from 5.1.x to 8.0.y

This causes the following issue. The hrls servlet will not load and gives the following error:
```
Caused by: com.mysql.cj.exceptions.InvalidConnectionAttributeException: The server time zone value 'CEST'
```
In version 8 it needs a timezone to be set: https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-other-changes.html
CEST is just not known.

This can be fixed by adding a timezone to the connection string for `handlereverselookupservlet.properties `. An example being:
```
sqlConnectionString = jdbc\:mysql\:///master_database?autoReconnect=true&serverTimezone=Europe/Amsterdam
```
Here following is added:
```
&serverTimezone=Europe/Amsterdam
```

--- a/README.md
+++ b/README.md
@@ -51,13 +51,17 @@ A typical configuration for EUDAT for Handle servers that use SQL storage may be
 ```
 useSql = true
 useSolr = false
-sqlConnectionString = jdbc\:\mysql\://localhost\:3306/database_name?autoReconnect=true
+sqlConnectionString = jdbc\:\mysql\://localhost\:3306/database_name?autoReconnect=true&serverTimezone=Europe/Amsterdam
 sqlUsername = user
 sqlPassword = password
 jdbcDriverClassName = com.mysql.jdbc.Driver
 ```
 
 Note the escaping of colon characters in the the sqlConnectionString.
+Note the setting of the timezone. If this is not set the servlet will not start and complain about
+```
+com.mysql.cj.exceptions.InvalidConnectionAttributeException
+```
 
 By default, the servlet does not log all queries. This can be enabled by including the following in the config file. The query log will be under the location {HANDLE_SVR}/logs/hrls-requests.log. 
 

--- a/pom.xml
+++ b/pom.xml
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>HandleReverseLookupServlet</groupId>
   <artifactId>HandleReverseLookupServlet</artifactId>
-  <version>1.0.5</version>
+  <version>1.0.6</version>
   <build>
     <sourceDirectory>src</sourceDirectory>
     <finalName>reverselookupservlet-${project.version}</finalName>

--- a/pom.xml
+++ b/pom.xml
@@ -58,22 +58,22 @@
   	<dependency>
   		<groupId>org.apache.logging.log4j</groupId>
   		<artifactId>log4j-api</artifactId>
-  		<version>2.9.1</version>
+  		<version>2.13.2</version>
   	</dependency>
   	<dependency>
   		<groupId>org.apache.logging.log4j</groupId>
   		<artifactId>log4j-core</artifactId>
-  		<version>2.9.1</version>
+  		<version>2.13.2</version>
   	</dependency>
   	<dependency>
   		<groupId>org.apache.logging.log4j</groupId>
   		<artifactId>log4j-slf4j-impl</artifactId>
-  		<version>2.9.1</version>
+  		<version>2.13.2</version>
   	</dependency>
   	<dependency>
   		<groupId>org.apache.logging.log4j</groupId>
   		<artifactId>log4j-web</artifactId>
-  		<version>2.9.1</version>
+  		<version>2.13.2</version>
   	</dependency>
   	<dependency>
   		<groupId>commons-logging</groupId>
@@ -83,7 +83,7 @@
   	<dependency>
   		<groupId>mysql</groupId>
   		<artifactId>mysql-connector-java</artifactId>
-  		<version>5.1.38</version>
+  		<version>8.0.16</version>
   	</dependency>
   	<dependency>
   		<groupId>org.glassfish.jersey.media</groupId>

